### PR TITLE
Add User to the Login completion block

### PR DESCRIPTION
**Pull request by SkylerSeamans**

### CODE REVIEW

File: TeamSnapSDK/SDK/TSDKCompletionBlockTypes.h [hunk 1]
```diff
@@ -16,7 +16,8 @@ typedef void (^TSDKRootLinkCompletionBlock)(TSDKRootLinks *_Nullable rootLinks,
 typedef void (^TSDKCompletionBlock)(BOOL success, BOOL complete, TSDKCollectionJSON *_Nullable objects, NSError *_Nullable error);
 typedef void (^TSDKSaveCompletionBlock)(BOOL success, TSDKCollectionObject *_Nullable object, NSError *_Nullable error);
 typedef void (^TSDKInviteStatusCompletionBlock)(BOOL success, BOOL complete, TSDKinvitationFinder *_Nullable objects, NSError *_Nullable error);
-typedef void (^TSDKLoginCompletionBlock)(BOOL success, NSString *_Nullable OAuthToken, NSError *_Nullable error);
+typedef void (^TSDKTokenCompletionBlock)(BOOL success, NSString *_Nullable OAuthToken, NSError *_Nullable error);
+typedef void (^TSDKLoginCompletionBlock)(BOOL success, NSString *_Nullable OAuthToken, TSDKUser *_Nullable user, NSError *_Nullable error);
```
Comment (jrahaim) [inline TeamSnapSDK/SDK/TSDKCompletionBlockTypes.h]: @SkylerSeamans 
>  [[TSDKTeamSnap sharedInstance] loginWithOAuthToken:OAuthToken completion:^(BOOL success, NSError *error, TSDKUser *user)

I think user / Error are swapped between this, and root links #366

**Comment by jrahaim** [comment]
The ordering seems inconsistent on when it's `...,user, error` and `...,error, user`

**Comment by SkylerSeamans** [comment]
@jrahaim updated

**Comment by jrahaim** [comment]
:+1: tests pass
:+1: Code looks good. 
Have not tested yet.

**Comment by SkylerSeamans** [comment]
Waiting to merge until the app is updated

**Pull request merged**